### ISSUE TITLE
BOM for non-printed parts? 

### ISSUE THREAD

**Issue by Darkhand81** [0x23/QuietAirCompressor#1]
Awesome project! Do you have a BOM or list of the non-printed parts, such as the quantity and dimensions of the rods, screws, etc, as well as where you sourced the pistons, cylinders, and valvehead?

**Comment by 0x23** [comment]
No, unfortunately I dont have a BOM but you can lookup all the dimensions
in the CAD file and if you like to contribute create a BOM for the project
yourself.
The Pistons and Cylinders were taken from this cheap car tire air
compressor:

[image: epap03-air-pump.jpg]
But be warned - the thing is really low quality (The cooling fins are fake
and made from Plastik) and it did not seal very well making it unusable for
low RPM.
Thats why I have only taken the seals and the Cylinderpipes and discarded
the rest.

**Comment by KKobel** [comment]
I have been compiling a 'BOM' for the purchased parts.
The following are Amazon ASIN numbers:
Connecting Rod Bearing: B07TJFK68V
Main Bearing: B07FDWW9D9
Pullys and Belt: B0D97C6HFJ
M3x10: B01MFA9YEP 
M3x16: B077XVVWYW
M3 Flat Washer: B015A39NCC 
M4 Threadall: B09LH7PX55
M4 nuts: B0CMC61C9D
2 & 3 mm dowel pins: B0BKZND42Z

Please verify these.

What I am having trouble with is the piston seal.
Can anyone help with that.

**Comment by 0x23** [comment]
I used the seals and the cylinder pipes from a cheap air compressor. The seals look very similar to something like this: B07ZSB5ZFC (not correct size, just an example). Such seels are very common in hydraulics and if possible I would lean towards a softer seal material (NBR). Maybe this helps.

**Comment by KKobel** [comment]
Thanks,

I found this:
https://www.mcmaster.com/9505K22/
Interestingly, the 'loaded' seals are cheaper than the 'unloaded' ones.

BTW - I am making the cylinder and head from scratch and using and external check valve on the output.
I'm using a DC brushed motor. The length is perfect. I had to modify the brackets.
I 3D printed all the parts for a fit-up.
So far, so good.

My use case is a 1.5" scale RS3 for the air brakes.

Again, thanks for a great project.
Karl

**Comment by 0x23** [comment]
Nice, It would be cool If you send me some pictures of compressor when its
done. Also make sure to minimize the volume between the cyclinder in the
up-position and the (external) checkvalves.
Any volume there will reduce the compression ratio and thus the maximum
pressure you can reach.

Good luck with cour project!

**Comment by KKobel** [comment]
Thanks for the heads-up on the check valve.
I probable would have had to learn that the hard way ;)

I'll take some pictures when I get closer to finishing.

Karl

**Comment by KarlJKobel** [comment]
Can you send me a link to the cheap compressors? McMaster doesn't seem to have the right seal.

**Comment by tomato7722** [comment]
Hello people,
I just started this projekt, where you guys able to finish it? i would love some updates on this almost dead thread..
Did this ( https://www.mcmaster.com/9505K22/ ) seal fit good?
Also im too stupid to find the right size motors. Got some 3,5 nm steppers and they are waay to large.
I lucky have access to a mill so the zylinder an zylinderheads where no problem.

I guess ill update this when i get further down the line...

**Comment by KarlJKobel** [comment]
I got mine working. I had no luck with the 3D printed pistons and seals. I bought 2 cheap storm door closers and used the tube for the cylinders and adapted the pistons to the connecting rods. The seal is a simple O ring and there is a ramp on the piston that makes the seal. I can only get 40 pounds, but that is good enough for my application. I suspect the 3D printed cylinder heads are porous and or leaking. I may deal with that in the future. I'll have to take some pictures and post them.
